tests/trainer/test_checkpoint.py

```diff
@@ -1696,6 +1696,38 @@ def test_set_dataloaders_to_cur_epoch(
         # Epoch count starts at O
         assert trainer.state.train_dataloader.batch_sampler.epoch == max_duration - 1
 
+    @world_size(2)
+    @pytest.mark.gpu
+    def test_load_incorrect_path(self, world_size: int, tmp_path: pathlib.Path, caplog):
+        save_folder = tmp_path / 'checkpoints'
+        save_folder.mkdir(exist_ok=True)
+
+        # Train and save a checkpoint
+        trainer = self.get_trainer(
+            save_folder=str(save_folder),
+            save_filename='checkpoint.pt',
+            save_interval='1ep',
+            max_duration='1ep',
+            device='gpu',
+        )
+        trainer.fit()
+        trainer.close()
```

Comment:
this isn't actually needed for test right?

Reply:
removed